### PR TITLE
localization for subscribe button

### PR DESCRIPTION
**Pull request by Arhell**
issue #21618
en [PR ](https://github.com/kubernetes/website/pull/20878)

/language it

### CODE REVIEW

**Comment by Arhell** [comment]
review page
https://deploy-preview-21692--kubernetes-io-master-staging.netlify.app/it/#kubeweekly

File: i18n/it.toml [hunk 1]
```diff
@@ -178,6 +178,9 @@ other = "Prima di cominciare"
 [ui_search_placeholder]
 other = "Cerca"
 
+[subscribe_button]
```
Comment (kbhawkey) [inline i18n/it.toml:181]: Hi @Arhell .
What do you think about adding the `subscribe_button` field before the `ui_search_placeholder` field?
The fields are listed alphabetically.

Comment (Arhell) [reply i18n/it.toml:178]: Hi, done, thanks

**Comment by kbhawkey** [comment]
LGTM

**Comment by Fale** [comment]
I think "Iscriviti" would be a better translation

**Comment by Arhell** [comment]
@Fale 
Done, thanks

**Comment by Fale** [comment]
/lgtm
(probably it should be squashed, I guess)

**Comment by Arhell** [comment]
@Fale
> (probably it should be squashed, I guess)

Done

**Comment by rlenferink** [comment]
/assign @kbhawkey 
for approval (file in i18n directory)

**Comment by sftim** [comment]
Echoing https://github.com/kubernetes/website/pull/21692#pullrequestreview-430203405
/approve

**Pull request merged**